### PR TITLE
EDU-992 Add external account option on login page

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-992

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ The output of this repository is an npm package (`@educandu/educandu`).
...
- | ambConfig | Configuration for the AMB endpoint (https://dini-ag-kim.github.io/amb/) | `{ apiKey: <string>, image: <string>, publisher: [{ type: <'Organization'/'Person'>, name: <string> }] }, about: [{ id: <category URL from https://skohub.io/dini-ag-kim/hochschulfaechersystematik/heads/master/w3id.org/kim/hochschulfaechersystematik/scheme.en.html>}]` | no, however if provided, `apiKey` is mandatory |
+ | ambConfig | Configuration for the AMB endpoint (https://dini-ag-kim.github.io/amb/) | `{ apiKey: <string>, image: <string>, publisher: [{ type: <'Organization'/'Person'>, name: <string> }], about: [{ id: <category URL from https://skohub.io/dini-ag-kim/hochschulfaechersystematik/heads/master/w3id.org/kim/hochschulfaechersystematik/scheme.en.html>}] }` | no, however if provided, `apiKey` is mandatory |
+ | externalAccountProviders | Configuration for SAML authentication | `{ key: <string>, displayName: <string>, loginUrl: <string>, logoUrl: <string>, expiryTimeoutInDays: <number> }` | no, however if provided, all fields except `logoUrl` and `expiryTimeoutInDays` are mandatory |
```
Comment (adrianaluchian) [inline README.md:54]: We could mention here that `[...] are mandatory, and 'expiryTimeoutInDays' defaults to 180 days`

File: src/components/pages/login.js [hunk 2]
```diff
@@ -49,8 +76,20 @@ function Login({ PageTemplate, SiteLogo }) {
               </div>
             </Fragment>
           )}
+          {clientConfig.externalAccountProviders.length && (
```
Comment (adrianaluchian) [inline src/components/pages/login.js:79]: `!!clientConfig.externalAccountProviders.length` 😬

File: README.md (hunk 1 above)
Comment (ahelmberger) [reply README.md]: Sure, updated!

**Comment by adrianaluchian** [comment]
Looks super nice! 😁 Actually better than a dropdown with options 👍

File: src/components/pages/login.js (hunk 2 above)
Comment (adrianaluchian) [reply src/components/pages/login.js]: 😄 👍

**Review by adrianaluchian — APPROVED** [review]

**Comment by ahelmberger** [comment]
> Looks super nice! grin Actually better than a dropdown with options +1

:bow:

**Pull request merged**